backend/static/frontend/org/riceapps/controllers/schedule_planner_controller.js

```diff
@@ -399,7 +420,8 @@ SchedulePlannerController.prototype.start = function() {
   this.getHandler().
     listen(this.view_, DraggableView.EventType.CLICK, this.onCourseViewClick_).
     listen(this.view_, DraggableView.EventType.DROPPED, this.onCourseViewDropped_).
-	listen(this.view_, DraggableView.EventType.CLICK, this.onCRNViewClick_).
+	  listen(this.view_, DraggableView.EventType.CLICK, this.onCRNViewClick_).
+    listen(this.view_, DraggableView.EventType.CLICK, this.onClearPlaygroundClick_).
```

Comment:
You should be using goog.events.EventType.CLICK instead of DraggableView.EventType.CLICK as you are listening for a DOM element click and not a click on a DraggableView.